test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml

```diff
@@ -1,16 +1,16 @@
 data_source = "AEO2021 Reference Case"
 dataset_id = "aeo2021_ref_com_energy_end_use_growth_factors"
-data_schema_type = "one_table"
-dataset_type = "benchmark" # <--- ?
-path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Use Growth Factors/data/" # <---
+data_schema_type = "growth_rate"
+dataset_type = "benchmark"
```

Comment:
I'm a little torn about whether this is actually a benchmark dataset.

Reply:
We also have an End Use dataset in the test data from dsgrid PR #99, which I think is definitely benchmark. Let me know what you want for the growth rates. Keep in mind that this is the test data, but I am taking notes for when we'll register the real thing.